src/run.rs

```diff
@@ -215,31 +228,44 @@ impl Handler for ExecHandler {
         // 3. Just send a specified signal to the child, do nothing more
         // 4. Make sure the previous run was ended, then run the command again
```

Comment:
Can you also update this comment given there's now five scenarios

Reply:
Yeah, sorry, my bad..